wgengine/filter/match.go

```diff
@@ -117,3 +122,12 @@ func ipInList(ip netaddr.IP, netlist []netaddr.IPPrefix) bool {
 	}
 	return false
 }
+
+func protoInList(proto packet.IPProto, valid []packet.IPProto) bool {
```

Comment:
Feelings on keeping list vs. making a map? Assuming for short list the linear traversal will be best.

Reply:
I can do a map. A map has a linear scan fast path for small maps (<=8 IIRC) anyway.

Or at that point, I can do a bitfield like @sailorfrag said.

Reply:
Will do that tonight before submitting.... like `type protoMap [4]uint64`

Reply:
I'm also fine not over-optimizing out the door, a linear scan is nice and cache-friendly.

Reply:
Okay, will submit this as-is.

Then optimizations can come later with benchmarks justifying themselves.